### ISSUE TITLE
Support for default values from SQL functions

### ISSUE THREAD

**Issue by Danysan1** [kitloong/laravel-migrations-generator#96]
I am trying to create the migrations for a PostgreSQL DB which contains columns with the default value obtained with a SQL function.
This is an example of a creation script for an existing table:

```sql
-- ...
CREATE TABLE public.foo (
-- ...
    bar timestamp(6) without time zone DEFAULT timezone('Europe/Rome'::text, now()),
-- ...
```

Currently the migration generated with `php artisan migrate:generate` is:

```php
// ...
        Schema::create('foo', function (Blueprint $table) {
// ...
            $table->timestamp('bar', 6)->nullable()->default('Europe/Rome');
// ...
```

But this is not correct, as it implies the default value is the string 'Europe/Rome', not the function `timezone('TIMEZONE_NAME'::text, now())`.
In fact running this migration causes this error:

```sql
SQLSTATE[22007]: Invalid datetime format: 7 ERROR:  invalid input syntax for type timestamp: "Europe/Rome" at character 204 (SQL:
create table "foo" (
-- ...
  "bar" timestamp(6) without time zone null default 'Europe/Rome',
-- ...
```

If I manually substitute `'Europe/Rome'` with `DB::raw("timezone('Europe/Rome'::text, now())")` it works:

```php
// ...
        Schema::create('foo', function (Blueprint $table) {
// ...
            $table->timestamp('bar', 6)->nullable()->default(DB::raw("timezone('Europe/Rome'::text, now())"));
// ...
```

It would be cool if the library natively interpreted correctly default values from SQL functions

**Comment by kitloong** [comment]
Hi @Danysan1 

Thank you for reporting this issue, and providing valuable information for the details!
I understand the generator currently is generating the wrong default value with timezone, let me see how I can improve the generator for this

**Comment by Danysan1** [comment]
Thank you

**Comment by kitloong** [comment]
Released with tag `v6.3.0`
